brayns/Brayns.cpp

```diff
@@ -431,40 +405,26 @@ struct Brayns::Impl : public PluginAPI
                     break;
                 }
             }
-            pm.setProperty({"morphologySectionTypesSoma",
-                            "morphologySectionTypesSoma", soma});
-            pm.setProperty({"morphologySectionTypesAxon",
-                            "morphologySectionTypesAxon", axon});
-            pm.setProperty({"morphologySectionTypesDendrite",
-                            "morphologySectionTypesDendrite", dendrite});
-            pm.setProperty({"morphologySectionTypesApicalDendrite",
-                            "morphologySectionTypesApicalDendrite",
-                            apicalDendrite});
+            pm.addProperty({"sectionTypesSoma", "sectionTypesSoma", soma});
```

Comment:
what happend to 'fullDetailed', 'noAxon' and 'onlySomas'?

Reply:
For later, this commit is older than this discussion and I need it merged for the upcoming ones.